cms/templates/js/certificate-details.underscore

```diff
@@ -27,6 +27,10 @@
          </header>
          <div class="signatory-details-list"></div>
     <% } %>
+    <div class="certificate-activation">
+        <button id="activate-certificate" class="activate-cert <% if(!is_active) { %> active-state <% } else { %>de-active-state <% } %>"><span><%= gettext("Activate") %></span></button>
```

Comment:
If the final implementation is 2 separate buttons, +1 to @mattdrayer 's button text recommendations.